src/components/forms/DomainSettings/DomainSettings.tsx

```diff
@@ -113,22 +113,12 @@ function DomainSettings({
         refetchType: 'all',
       });
 
-      if (domain) {
-        queryClient.invalidateQueries({
-          queryKey: ['arns-record', { domain }],
-          refetchType: 'all',
-        });
-      }
-
-      if (data?.processId) {
-        queryClient.invalidateQueries({
-          queryKey: ['ant', data?.processId.toString()],
-          refetchType: 'all',
-        });
-      }
-
       queryClient.invalidateQueries({
-        queryKey: ['domainInfo'],
+        queryKey: ['domainInfo', data?.processId.toString(), domain],
```

Comment:
fix this cache invalidation

Reply:
[45a10f5](https://github.com/ar-io/arns-react/pull/666/commits/45a10f59a833ec0ccb4352db4cf412c7adbd8c87)